### DISCUSSION TITLE
Upgrading to Home Assistant 2023.5.0 broke this integration.

### DISCUSSION THREAD

**Discussion by DarrylGNOLA** [vwt12eh8/hassio-ecoflow#78] [General]
Today I upgraded to 
"Home Assistant 2023.5.0
Supervisor 2023.04.1
Operating System 10.1
Frontend 20230503.1 - latest"
which broke the Ecoflow integration. It seems others are having many issues with it breaking various connections as well.

I didn't have time to mess with it at the moment and I just did a restore of 2023.4.6.

**Comment by Patrick762** [comment]
There's already an issue about this #76 
If you use the stable version, you can test the change made in #79
